### PR TITLE
UIIN-2658: Switch from = to == operator when querying for holdings-storage

### PR DESCRIPTION
**Pull request by mkuklis**
https://issues.folio.org/browse/UIIN-2658

The `=` operator was taking too long to finish in Poppy.

This PR switched from `=` to `==` operator when querying for `holdings-storage/holdings` by hrids.

### CODE REVIEW

**Review by MikeTaylor — APPROVED** [review]
Yep!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Make Inventory search and browse query boxes expandable. Refs UIIN-2493.
 * Added support for `containsAny` match option in Advanced search. Refs UIIN-2486.
 * Inventory search/browse: Do not retain checkbox selections when toggling search segment. Refs UIIN-2477.
+* Switch from `=` to `==` operator when querying for `holdings-storage/holdings` by hrid. Fixes UIIN-2658.
```
Comment (mariia-aloshyna) [inline CHANGELOG.md:9]: If this should be released as a Poppy bug fix then move it to v10.0.3, please

Comment (mkuklis) [reply CHANGELOG.md]: @mariia-aloshyna I just moved it under `v11.0.3`

Comment (mariia-aloshyna) [reply CHANGELOG.md]: Hi @mkuklis looks like it was removed from `v11.0.0` but wasn't moved to `v10.0.3`

**Comment by MikeTaylor** [comment]
Probably needs a few more merge-from-master events before this is ready :-)

**Pull request merged**